From f993884dde6c09b12cc5ceb3d9df7b3896196e32 Mon Sep 17 00:00:00 2001
From: Ben Hilburn <ben.hilburn@ettus.com>
Date: Fri, 10 Mar 2017 15:34:17 -0500
Subject: [PATCH 04/72] Doxygen: Fixing license notice on Doxygen-generated
 `index.html`.

Also fixed old link to Tom's blog to point to new GNU Radio blog.
---
 docs/doxygen/other/main_page.dox | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/docs/doxygen/other/main_page.dox b/docs/doxygen/other/main_page.dox
index 67487ddfd6..8f28f23dc8 100644
--- a/docs/doxygen/other/main_page.dox
+++ b/docs/doxygen/other/main_page.dox
@@ -2,14 +2,14 @@
 
 \image html gnuradio-logo.svg
 
-Welcome to GNU Radio!
+<H1> Welcome to GNU Radio! </H1>
 
 For details about GNU Radio and using it, please see the
 <a href="http://gnuradio.org" target="_blank"><b>main project page</b></a>.
 
 Other information about the project and discussion about GNU Radio,
 software radio, and communication theory in general can be found at
-the <a href="http://www.trondeau.com" target="_blank"><b>GNU Radio blog</b></a>.
+the <a href="http://www.gnuradio.org/blog" target="_blank"><b>GNU Radio blog</b></a>.
 
 This manual is split into two parts: A usage manual and a reference. The usage manual
 deals with concepts of GNU Radio, introductions, how to build GNU Radio etc.
@@ -22,4 +22,13 @@ A search function is also available at the top right.
 \li \subpage page_usage "Part I - GNU Radio Usage"
 \li \subpage page_components "Part II - Reference"
 
+<H3> Documentation License </H3>
+
+The Free Software Foundation gives permission to use and distribute
+the program source documentation strings, after processing them
+through Doxygen or some comparable program, under the GNU Free
+Documentation License, version 1.3 or later.  However, we do not grant
+such permission for the program code in our release (aside from the
+documentation strings).
+
 */
-- 
2.11.0

